msticpy/sectools/tiproviders/kql_base.py

```diff
@@ -208,14 +208,14 @@ def lookup_iocs(  # noqa: C901, MC0001
                     and data_result.completion_query_info["StatusCode"] == 0
                     and data_result.records_count == 0
                 ):
-                    warnings.warn("No results return from data provider.")
+                    print("No results return from data provider.")
```

Comment:
This message reads strangely, should it be "No results returned from data provider"?